src/frontend/src/components/NavBar/components/RightNav/index.tsx

```diff
@@ -11,13 +19,33 @@ export default function RightNav(): JSX.Element {
     return (
       <StyledDiv>
         <Link href={ROUTES.UPLOAD_STEP1} passHref>
-          <a href="passHref">
-            <UploadButton sdsType="secondary" sdsStyle="rounded">
-              Upload
-            </UploadButton>
-          </a>
+          <UploadButton
+            component="a"
```

Comment:
We no longer need to have a button inside of a link, we can use the component prop instead so the button style uses the `a` tag.